libs/sdk-utils/state-listener/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "root": "libs/sdk-utils/state-listener",
+  "sourceRoot": "libs/sdk-utils/state-listener/src",
+  "projectType": "library",
+  "targets": {
+    "lint": {
```

Comment:
Build? 👀

Reply:
this will maybe be used in the future to configure a bundle